packages/lambda-tiler/src/routes/fonts.ts

```diff
@@ -43,6 +48,8 @@ export async function fontList(): Promise<LambdaHttpResponse> {
   const assetLocation = Env.get(Env.AssetLocation);
   if (assetLocation == null) return NotFound;
 
+  if (assetLocation.endsWith('.tar.co')) return serveFromCotar(assetLocation, 'fonts.json', 'application/json');
```

Comment:
I like this, then we will not broken the existing ones until the new one working very well.

Reply:
it also means we can still use local files for assets :)